### PR TITLE
Fix broken links for pachyderm application

### PR DESCRIPTION
**Pull request by ashley-o0o**
Closes: [RHOAIENG-10901](https://issues.redhat.com/browse/RHOAIENG-10901)

## Description
Fixes broken links for the dashboard application Pachyderm

## How Has This Been Tested?
Tested locally on cluster

## Test Impact
no new tests

## Request review criteria:

Self checklist (all need to be checked):
- [x] The developer has manually tested the changes and verified that the changes work
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has added tests or explained why testing cannot be added (unit or cypress tests for related changes)

If you have UI changes: 

- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change.

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

**Review by jpuzz0 — APPROVED** [review]
Verified all new links are working properly.

File: manifests/rhoai/shared/apps/pachyderm/pachyderm-app.yaml
Comment (DaoDaoNoCode) [inline manifests/rhoai/shared/apps/pachyderm/pachyderm-app.yaml:1]: I opened a similar ticket before https://issues.redhat.com/browse/RHOAIENG-10207 and confirms that we still have a broken link on line 21 https://github.com/opendatahub-io/odh-dashboard/pull/3240/files#diff-78e76e77265d57ee5058fdab8c1e9bbebc51012a5e1044dc1dcc90201202d251R21. This is the Get started button on the side panel. Please coordinate with the team and find the best link for that field.

Screenshot 2024-09-23 at 4 47 03 PM

Comment (christianvogt) [reply manifests/rhoai/shared/apps/pachyderm/pachyderm-app.yaml:1]: Looks like this should be updated to: https://docs.pachyderm.com/products/mldm/latest/get-started/

**Review by DaoDaoNoCode — APPROVED** [review]
/lgtm

**Pull request merged**